packages/@uppy/transloadit/src/index.js

```diff
@@ -779,7 +778,7 @@ module.exports = class Transloadit extends Plugin {
         // were added to the Assembly, so we can properly complete it. All that state is handled by
         // Golden Retriever. So, Golden Retriever is required to do resumability with the Transloadit plugin,
         // and we disable Tus's default resume implementation to prevent bad behaviours.
-        resume: false,
+        storeFingerprintForResuming: false,
```

Comment:
The resume option is removed but you want the storeFingerprintForResuming option here if I understand the comment above correctly.